placetobe-header.html

```diff
@@ -235,10 +239,20 @@
         menuOpen: {
           type: Boolean,
           value: false
+        },
+
+        campaignArrayHidden: {
+          type: Boolean,
+          value: true,
+          computed: '_checkCampaignArrayLength(apiCampaignData)'
```

Comment:
`computeHasCampaigns(apiCampaignData.splices)`

want anders checkt ie niet of er array items bij komen en werkt het alleen de eerste keer